app/components/SortableTableHeader.tsx

```diff
@@ -16,13 +16,20 @@ const SortableTableHeader: React.FunctionComponent<Props> = ({
 }) => {
   const {columnName, displayName} = headerVariables;
   return (
-    <th
-      style={{background: '#003366', color: 'white'}}
-      onClick={() => sort('sortColumn', columnName)}
-    >
-      {displayName}
-      <br />
-      {displayName && <FontAwesomeIcon color="white" icon={faSort} />}
+    <th onClick={() => sort('sortColumn', columnName)}>
+      <span>{displayName}</span>
+      <span style={{height: '100%', position: 'absolute', right: '0.75em'}}>
```

Comment:
Doesn't a flexbox work, instead of using absolute/relative positioning?

Reply:
Positioning the icon independent of the box model flow was needed in order to maintain its flush-right positioning regardless of label text or viewport width. Flex layout was apt to place it on the following line next to the last text character, among other problems. 

Basically: fluid behavior of the icon is undesirable in this case. Only too happy to accept tweaks if you'd like to take a crack at it yourself though :)